uw-research-computing/gpu-jobs.md

```diff
@@ -217,6 +221,13 @@ attributes shown above. If you want a certain class of GPU, use CUDACapability:
 	code so that it can run across GPU types and without needing the
 	latest version of CUDA.
 
+- **Specify Multiple Requirements (optional)**: Multiple requirements can be specified by using && statements:
+	```
+	requirements = (CUDACapability == 7.5) && (CUDADeviceName = "GeForce RTX 2080 Ti")
```

Comment:
What do you think about changing this example to require some min or max CUDA capability and some minimum amount of GPU RAM?  We generally encourage users to not request a specific type of GPU because that can constrain where their jobs can run more than is necessary.

Reply:
Sounds good! I checked using condor_status available attributes for specifying GPU memory. CUDAGlobalMemoryMb is an attribute that can be specified I can push changes.